settings/com.endlessm.settings.gschema.override.in

```diff
@@ -39,7 +39,7 @@ button-layout=':minimize,maximize,close'
 
 # Specify the default background image
 [org.gnome.desktop.background]
-picture-uri='eos:///default'
+picture-uri='file:///usr/share/eos-media/desktop-background-C.jpg'
```

Comment:
@rshuler I think I would like to keep all the background images in eos-media. A good solution would probably be splitting an `eos-background-default` debian package from eos-media and require that from both the installer and the full image, with the full image also requiring the full eos-media package.
Will work on that Monday.